readme.md

```diff
@@ -195,13 +196,29 @@ find my_folder -mtime +5
 
 ## <a id="showcal"></a>show a calendar
 
-**STOP LOOKING UP WHAT DAY IT IS BY CALENDAR WIDGETS**
+**STOP LOOKING UP WHAT THIS MONTH LOOKS LIKE BY CALENDAR WIDGETS**
 
-Display a text calendar, highlighting the current day
+Display a text calendar
 
 ```
 cal
 ```
 
+## <a id="calcdate"></a>find a future date
+
+**STOP USING WEBAPPS TO CALCULATE FUTURE DATES**
+
+What is todays date?
+
+```
+date +%m/%d/%Y
+```
+
+What about a week from now?
+
+```
+date -d "+7 days"
```

Comment:
hmmm, from [this link](http://stackoverflow.com/questions/1706882/get-the-date-a-day-before-current-time-in-bash/8521717#8521717) it looks like Linux uses GNU Date, but OSX uses a different date function?

Does this work?
```
date -j -v+7d
```